pkg/p2p/kadcast/peer_test.go

```diff
@@ -80,7 +80,7 @@ func TestListenStreamReader(t *testing.T) {
 		b := m.Payload().(block.Block)
 		assert.True(m.Category() == topics.Block)
 		assert.True(b.Header.Height == 5525)
-		assert.True(len(b.Txs) == 12)
+		assert.True(len(b.Txs) == 11)
```

Comment:
Wut?

Reply:
What do you mean by `Wut`? If you read the PR in depth, you'll notice that unit test ` TestListenStreamReader` uses createBlockMessage which calls `RandomBlock`. With this PR we remove `RandDistributeTx` from `RandomBlock` because it's only Rusk that knows how to generate it.